spec/requests/sessions_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
```

Comment:
The spec should be under the folder spec/controllers/sessions_controller_spec.rb

Reply:
modified this